packages/skills/weather_station/dummy_weather_station_data.py

```diff
@@ -87,11 +112,10 @@ def main(self):
             dict_of_data['wind_ave'] = random.randrange(0, 10, 1)
             dict_of_data['wind_dir'] = random.randrange(0, 14, 1)
             dict_of_data['wind_gust'] = random.randrange(1, 7, 1)
-            print(dict_of_data)
-            self.addData(dict_of_data)
+            self.add_data(dict_of_data)
             time.sleep(5)
 
 
 if __name__ == '__main__':
```

Comment:
@Totoual rather than creating these manually upfront we could create these in a task as part of the skill i.e. each call to task `execute()` causes a call to `generate` (with the while loop removed)